package.json

```diff
@@ -23,7 +23,6 @@
   },
   "devDependencies": {
     "@types/node": "^20.11.16",
-    "@vitest/coverage-c8": "^0.33.0",
```

Comment:
I don't think the workspace-wide coverage reporting has worked for awhile, but just to check - does this change behavior of what's reported anywhere else?

Reply:
As far as I could tell, the behavior looks to be the same without this package. There's a deprecation notice at the top of the package page on the npm registry - https://www.npmjs.com/package/@vitest/coverage-c8 stating that "v8 coverage is moved to @vitest/coverage-v8 package," which is also installed as a dependency.